domain/flows/order_pizza.yml

```diff
@@ -0,0 +1,149 @@
+version: "3.1"
+
+intents:
+  - greet
+  - inform_pizza
+  - inform_address
+  - inform_num_pizza
+  - ask_availability
+  - correct_order
+  - correct_num_pizza
+  - correct_pizza_type
+  - correct_payment_option
+  - affirm
+  - deny
+
+entities:
+  - pizza_type
+  - address
+  - department
+
+slots:
+  pizza:
+    type: text
+    mappings:
+        - type: from_entity
+          entity: pizza_type
+  num_pizza:
+    type: float
+    mappings:
+        - type: from_entity
+          entity: number  # this is defined in the restaurant.yml domain file
+          intent: inform_num_pizza
```

Comment:
you could also use duckling here as it is running anyway

Reply:
Yep it's in use. I only added a comment to indicate where the entity is defined so I don't add a duplicate entry in this domain file.